Altis_Life.Altis/core/pmenu/fn_useItem.sqf

```diff
@@ -6,10 +6,21 @@
     Description:
     Main function for item effects and functionality through the player menu.
 */
-private "_item";
 disableSerialization;
 if ((lbCurSel 2005) isEqualTo -1) exitWith {hint localize "STR_ISTR_SelectItemFirst";};
-_item = CONTROL_DATA(2005);
+private _item = CONTROL_DATA(2005);
+
+if !(M_CONFIG(getNumber,"VirtualItems",_item,"edible") isEqualTo -1) exitWith {
+    if ([false,_item,1] call life_fnc_handleInv) then {
+        private _val = M_CONFIG(getNumber,"VirtualItems",_item,"edible");
```

Comment:
Thanks for the PR @BlackFisch. Looks mostly good, but just for my sanity can we avoid doing `M_CONFIG(getNumber,"VirtualItems",_item,"edible")` twice, so just define the var before, then check if the var isEqualTo -1 etc